README.md

```diff
@@ -127,7 +120,7 @@ Click the `Run Tests` button.
 - add these properties to your `gradle-local.properties`
   - bintray_user
   - bintray_key
-- change the `pkg_version` property in `gradle.properties` to the new version number
+- change the `version` property in `gradle.properties` to the new version number
 - `gradle bintrayUpload`
```

Comment:
might as well change to `./gradlew bintryUpload` for consistency

Reply:
Done